ansible_collections/arista/avd/molecule/eos_designs_negative_unit_tests/inventory/host_vars/connected-endpoints-mismatched-descriptions.yml

```diff
@@ -0,0 +1,20 @@
+loopback_ipv4_pool: 192.168.0.0/24
+
+type: l2leaf
+l2leaf:
+  defaults:
+  nodes:
+    - name: connected_endpoints
```

Comment:
wrong node names if you compare to the inventory: `connected-endpoints-mismatched-descriptions`